Author: Ricardo Mones <mones@debian.org>
Subject: fix typo in manpage spotted by lintian
 Cleans lintian I: typo-in-manual-page retrive retrieve
 [usr/share/man/man1/joy2key.1.gz:28]
Last-Update: 2022-04-17
Forwarded: no

diff --git a/joy2key.1 b/joy2key.1
index 6ed76a7..10959cd 100644
--- a/joy2key.1
+++ b/joy2key.1
@@ -25,7 +25,7 @@ note: [] denotes `optional' option or argument,
 .I joy2key 
 monitors the joystick (normally /dev/input/js0) and sends keyboard events to
 an X window, terminal, or raw console.  In the case of X, xwininfo(1)
-is used to retrive the window ID, which should not be a problem as
+is used to retrieve the window ID, which should not be a problem as
 this is a standard utility in most X11 distributions.  To find the
 symbolic name of a particular key, see include/X11/keysymdef.h.  In
 terminal mode, joy2key sends the ascii character given on the command
